docs/product/components/inputs.html

```diff
@@ -131,7 +131,7 @@
     {% header "h4", "Warning" %}
     <div class="stacks-preview">
 {% highlight html %}
-<div class="d-flex gs4 gsy fd-column has-warning">
+<div class="d-flex gy4 fd-column has-warning">
```

Comment:
### Changes requested

We should determine how we want to handle `p.s-input-message` both in the example code blocks as well as the rendered example. I think adding `.mb0` to the paragraph element will suffice.

Reply:
👍